package.json

```diff
@@ -1,116 +1,116 @@
 {
-    "name": "@abusix/hailstorm",
```

Comment:
seems like storybook formatted the package.json, so you'll have to manually run prettier here